src/ontology/cob-edit.owl

```diff
@@ -1347,6 +1366,58 @@
         <obo:IAO_0000115>A process during which an organism comes into contact with another entity.</obo:IAO_0000115>
         <rdfs:label xml:lang="en">exposure of organism</rdfs:label>
     </owl:Class>
+
+
+
+    <!-- http://purl.obolibrary.org/obo/COB_0001100 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0001100">
+        <owl:equivalentClass>
+            <owl:Class>
+                <owl:unionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/COB_0001101"/>
+                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/COB_0001102"/>
+                </owl:unionOf>
+            </owl:Class>
+        </owl:equivalentClass>
+        <rdfs:comment>This class groups the two defined classes &quot;disease&quot; and &quot;disorder&quot;. It supports use cases that conflate the two classes for pragmatic reasons.</rdfs:comment>
+        <rdfs:label xml:lang="en">disease or disorder</rdfs:label>
+        <rdfs:seeAlso rdf:resource="https://github.com/OBOFoundry/COB/pull/226"/>
+        <rdfs:seeAlso>https://github.com/OBOFoundry/COB/issues/19</rdfs:seeAlso>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/COB_0001101 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0001101">
+        <obo:IAO_0000115>A disposition (i) to undergo pathological processes that (ii) exists in an organism because of one or more disorders in that organism.</obo:IAO_0000115>
+        <dc:creator rdf:resource="https://orcid.org/0000-0002-7356-1779"/>
+        <oboInOwl:hasRelatedSynonym>disorder</oboInOwl:hasRelatedSynonym>
+        <rdfs:label xml:lang="en">disease</rdfs:label>
+    </owl:Class>
+    <owl:Axiom>
+        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/COB_0001101"/>
+        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
+        <owl:annotatedTarget>A disposition (i) to undergo pathological processes that (ii) exists in an organism because of one or more disorders in that organism.</owl:annotatedTarget>
+        <obo:IAO_0000119 rdf:resource="http://purl.obolibrary.org/obo/ogms.owl"/>
+    </owl:Axiom>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/COB_0001102 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0001102">
+        <obo:IAO_0000115>A material entity which is clinically abnormal and part of an extended organism. Disorders are the physical basis of disease.</obo:IAO_0000115>
```

Comment:
In OGMS, which is aligned with BFO, disease is defined as "A disposition (i) to undergo pathological processes that (ii) exists in an organism because of one or more disorders in that organism." This definition comes from [PMID:21347182](https://pubmed.ncbi.nlm.nih.gov/21347182/), "Toward an Ontological Treatment of Disease and Diagnosis," 2009, by Scheuermann, Ceusters, and Smith.  This is original source of this definitions (not DO). This would be good reading for MONDO folks and other who are interested in the source of this debate.

Reply:
We have obviously read this

Reply:
(I know you have)